src/table/styles.js

```diff
@@ -126,14 +126,15 @@ export const VisuallyHidden = styled('span')({
 // ---------- TableTotals ----------
 
 export const TotalsCell = styled(TableCell, {
-  shouldForwardProp: (prop) => prop !== 'headerStyle' && prop !== 'isTop',
-})(({ headerStyle, isTop }) => ({
-  ...headerStyle,
+  shouldForwardProp: (prop) => prop !== 'isTop' && prop !== 'headRowHeight' && prop !== 'totalsStyle',
+})(({ totalsStyle, isTop, headRowHeight }) => ({
+  ...totalsStyle,
   fontWeight: 'bold',
   position: 'sticky',
   borderWidth: isTop ? '0px 1px 2px 0px' : '2px 1px 1px 0px',
```

Comment:
```suggestion
  borderWidth: isTop ? '0px 1px 1px 0px' : '1px 1px 1px 0px',
```

Reply:
The 2px is the border width that get thicker regarding where the total row shows up. It's for distinguishing between total row and body row.

Reply:
This is fine for now. Let's start making changes when UX has input